workspace-manager-clienttests/render-config.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 VAULT_TOKEN=${1:-$(cat "$HOME"/.vault-token)}
-WM_APP_SERVICE_ACCOUNT_VAULT_PATH=secret/dsde/terra/kernel/dev/dev/workspace/app-sa
+WM_APP_SERVICE_ACCOUNT_VAULT_PATH=secret/dsde/terra/kernel/integration/wsmtest/workspace/app-sa
```

Comment:
Do the client tests work with this change? (The GHA is broken, but you should still be able to run them manually).
The SA rendered from this path gets read in `/workspace-manager-clienttests/src/main/resources/serviceaccounts/dev-workspace-app-sa.json`, though I don't know if anything actually uses that credential

Reply:
They were broken earlier today. Still not working for me.

Reply:
(I wasn't on the VPN and saw the discussion of another issue giving 403s, so thought it was the same issue).